components/user-details.vue

```diff
@@ -171,6 +179,12 @@
           .get(`/accounts/users/${this.userId}`)
           .jsonApi()
           .flatten()
+
+        this.orders = await this.$hal()
+          .get('fulfillment/orders')
+          .parameter('filter[user-id]', this.userId)
```

Comment:
Why not filter for shipped complete here rather than iterating through all of their orders?